transform/operations.go

```diff
@@ -246,6 +246,28 @@ func (t *timeParse) transform(raw interface{}) (interface{}, error) {
 	return parsedTime.Format(t.Args["layout"]), nil
 }
 
+// stringToInteger is a transformOperation which takes a string and converts it into an Int
+type stringToInteger struct {
+}
+
+func (s *stringToInteger) init(args map[string]string) error {
+	return nil
+}
+
+func (s *stringToInteger) transform(raw interface{}) (interface{}, error) {
+	str, ok := raw.(string)
+	if !ok {
+		return nil, errors.New("stringToInteger only supports strings")
+	}
+
+	result, err := strconv.Atoi(str)
+	if err != nil {
+		return nil, errors.New(fmt.Sprintf("error converting to an Integer from a string: %s", err))
```

Comment:
if you're going to use fmt and errors you can probably just use fmt.Errorf()

Reply:
actually you probably don't need most of this

you could just 
`return strconv.Atoi(str)`